airbyte-test-utils/src/main/java/io/airbyte/test/airbyte_test_container/AirbyteTestContainer.java

```diff
@@ -155,10 +155,11 @@ private Consumer<OutputFrame> logConsumer(final String service, final Consumer<S
         final String message = prependService(service, log.replace("\n", ""));
         switch (c.getType()) {
           // prefer matching log levels from docker containers with log levels in logger.
-          case STDOUT -> LOGGER.info(message);
           case STDERR -> LOGGER.error(message);
           // assumption that this is an empty frame that connotes the container exiting.
           case END -> LOGGER.error(service + " stopped!!!");
+          // default includes STDOUT and anything else
+          default -> LOGGER.info(message);
```

Comment:
opinions on setting this as the default?

Reply:
That seems fine.  Worst case scenario is that we get a few extra cases logged to INFO.